resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb

```diff
@@ -223,7 +223,13 @@ fi
 <% end -%>
 
 %post
+# Unconditional post-install tasks
 %{_app_prefix}/scripts/install.sh postinst_redhat
+
+# Only run these on initial install, not ugrades
+if [ "$1" = "1" ]; then
```

Comment:
same questions about values and quotes

Reply:
redhat does this a little differently, but still a variable that lets you know whether this is an install/upgrade/etc. See the chart at https://fedoraproject.org/wiki/Packaging:Scriptlets. You need to quote it for the bash comparison